docs/.vuepress/head.js

```diff
@@ -1,4 +1,9 @@
 module.exports = [
+  [
+    'meta',
+    { name: 'viewport', content: 'width=device-width, initial-scale=1' }
+  ],
+  ['link', { rel: 'preconnect', href: 'https://www.google-analytics.com' }],
```

Comment:
Do we really want to preconnect if this resource is loaded async anyway and already non-blocking?

Reply:
this come from lighthouse verbatim

Reply:
ok then, I guess google loves been connected to everything ⚡